### ISSUE TITLE
Using with aurelia cli and bundling

### ISSUE THREAD

**Issue by wwwlicious** [aurelia-plugins/aurelia-plugins-tabs#2]
I'm having some trouble getting this plugin working with bundling and my loader config.

`aurelia.use.plugin("aurelia-plugins-tabs")` doesn't work as the loader is configured to check the vendor bundle. It returns require script errors

When adding the plugin to the vendor bundle (or app bundle), it cannot pick it up automatically as the amd folder doesn't contain an index.js file so I have to specify the bundle main manually.

```
{
  "name": "aurelia-plugins-tabs",
  "path": "../node_modules/aurelia-plugins-tabs/dist/amd",
  "main": "aurelia-plugins-tabs"
}

```
once I do that, I get no errors if I remove the `aurelia.use.plugin...`, but it also isn't picking up the templates. I can see the model is bound to the element though. 

```
<aup-tabs tabs.bind="myTabs" au-target-id="8"></aup-tabs>
```

If I add the `aurelia.use.plugn...` back in with the bundle configured, I get `plugin.load is not a function` errors. 

Chasing my tail here a bit and no doubt something I'm doing incorrectly but any idea what that might be?

**Comment by bartlenaerts** [comment]
Hi @wwwlicious,
like you said, it should be sufficient to add the following the aurelia.json:

```
{
  "name": "aurelia-plugins-tabs",
  "path": "../node_modules/aurelia-plugins-tabs/dist/amd",
  "main": "aurelia-plugins-tabs"
}
```

But you need also the set the value of `stub` to `false` in aurelia.json.

Grtz, Bart

**Comment by wwwlicious** [comment]
@bartlenaerts That was it, thanks!

**Comment by jfstephe** [comment]
@bartlenaerts - Hi, I'm using SystemJs rather than the aurelia cli and I was wondering if you had a SystemJs configuration equivalent of the above as I have the same problem with the plugin when bundling.
I've tried defining an 'aurelia-plugins-tab' package but I get 404 for the other tab files (e.g. 'Cannot GET /jspm_packages/npm/aurelia-plugins-tabs@2.5.2/aurelia-plugins-tabs-tabs.js')
Any ideas?
Thanks,
John

**Comment by jfstephe** [comment]
Hi,
I managed to fix this after a night and morning of intense googling.
I had to add the following (the last 2 are the extra unexpected items) to my bundle config, 'includes' section:

`
       "aurelia-plugins-tabs",
        "aurelia-plugins-tabs/**/*.js",
        "aurelia-plugins-tabs/**/*.html!text",
`

I also added 
`
    "aurelia-plugins-tabs": {
      "main": "aurelia-plugins-tabs.js"
    }
`
to the config.js packages section but I'm not sure if that's needed or not atm.

@bartlenaerts - On the assumption that you confirm the above is correct, can we get this added to the readme to help others in the future?

Hope this helps someone!
John
